connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/utils/ProtobufOptions.scala

```diff
@@ -38,6 +38,14 @@ private[sql] class ProtobufOptions(
 
   val parseMode: ParseMode =
     parameters.get("mode").map(ParseMode.fromString).getOrElse(FailFastMode)
+
+  // Setting the `recursive.fields.max.depth` to 0 allows the field to be recurse once,
```

Comment:
'0' disables recursion right? Why once? This might be difference in terminology. Thats why giving a quick example is better. Could you add this example?:

 Consider a simple simple recursive proto 'message Person { string name = 1; Person bff = 2}

> What would be spark schema when recursion 0, 1, and 2? I think : 

  - 0: struct<name: string, bff: null>
  - 1: struct<name string, bff: <name: string, bff: null>>
  - 2: struct<name string, bff: <name: string, bff: struct<name: string, bff: null>

Reply:
@rangadi Thank you for your suggestion. I have implemented it by adding a comment and a unit test to make the example more clear to users.